src/go/k8s/config/rbac/bases/operator/role.yaml

```diff
@@ -77,6 +77,30 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
@RafalKorepta with your changes to the operator helm chart, what do you think will be the best way to keep this and the helm chart in sync?

Reply:
One option would be to move operator helm chart to redpanda-operator repo. Any step in Github action could be borrowed from helm chart repo that builds/generates schema, partials and gotohelm. The roles could be translated strait from controller gen annotation to our go based files. Then gotohelm would generate our special templates. 

Still it's weird interaction. Maybe using `embed` of what controller-gen produces would get what we are looking for? The gotohelm would need to understand how embed yaml representation should be translated to helm template. 

I'm open to brain storm more alternatives.